pgmq-extension/sql/pgmq--1.4.4--1.5.0.sql

```diff
@@ -0,0 +1,131 @@
+CREATE OR REPLACE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)
```

Comment:
I don't think we need to add `pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)` to the migration script, since this one is not changing, right? Only `pgmq.drop_queue(queue_name TEXT)` needs to be in the migration.

Reply:
Oh I see, I'm wrong (default value is being dropped). We probably need to use `drop function` instead of `create or replace`

https://github.com/tembo-io/pgmq/actions/runs/11413295614/job/31774992193?pr=319

```
ERROR:  cannot remove parameter defaults from existing function
HINT:  Use DROP FUNCTION drop_queue(text,boolean) first.
```

Reply:
Should be fixed now.